src/ai_config/geminiConfig.js

```diff
@@ -11,12 +11,15 @@ const configFilePath = path.join(os.homedir(), '.gimme_readme_config');
 dotenv.config({ path: configFilePath });
 
 // Initialize Google Generative AI client
-const genAI = new GoogleGenerativeAI(process.env.GEMINI_KEY);
 
 // Export function to handle Gemini-specific prompting
-export async function promptGemini(prompt, model, temperature = 0.5) {
+export async function promptGemini(prompt, model, temperature = 0.5, apiKey) {
   try {
     // Dynamically initialize the Gemini model based on user input
+    // issue-31 dynamic get apikey from .toml config
```

Comment:
Over here, we can directly read the value of the `GEMINI_KEY`. However, since we are using `TOML` styled syntax, it would be better to use a library like `toml` to parse the `.gimme_readme` file which can be found in the user's home directory.

Reply:
I don't think I'm understand correctly? , the config.toml is seperated from .gimme_readme_config right? 

why would we care about .gimme_readme_config?